src/annotator/anchoring/html.ts

```diff
@@ -23,18 +25,20 @@ async function querySelector(anchor, options = {}) {
  * It encapsulates the core anchoring algorithm, using the selectors alone or
  * in combination to establish the best anchor within the document.
  *
- * @param {Element} root - The root element of the anchoring context.
- * @param {Selector[]} selectors - The selectors to try.
- * @param {object} [options]
- *   @param {number} [options.hint]
+ * @param root - The root element of the anchoring context.
+ * @param selectors - The selectors to try.
```

Comment:
Man, you must be entirely glad you pinged me as a reviewer to get such comment gold as this one!

A convention I follow is that I only use periods (`.`) if there is a full sentence involved. e.g.

```
@param root - The root element of the anchoring context
@param complexity - The amount of `complexity` this comment brings without
  adding any value. This can be said to be a high number.
```